### PR TITLE
add & pull s02 datasets from coda

### PR DESCRIPTION
**Pull request by IsaackMwenda**

### CODE REVIEW

File: run_scripts/1_kakuma_coda_get.sh [hunk 1]
```diff
@@ -36,6 +44,9 @@ DATASETS=(
     "kakuma_unmarried_fathers_community_view"
     "kakuma_lessons_learnt"
     "kakuma_show_suggestions"
+
+    "kakuma_community_views_on_girls_education"
+
```
Comment (as2388) [inline run_scripts/1_kakuma_coda_get.sh]:
```suggestion
```

File: run_scripts/3_generate_outputs.sh [hunk 2]
```diff
@@ -32,7 +32,11 @@ GOOGLE_CLOUD_CREDENTIALS_FILE_PATH=$2
 PIPELINE_CONFIGURATION_FILE_PATH=$3
 DATA_ROOT=$4
 
-mkdir -p "$DATA_ROOT/Coded Coda Files"
+if [ -d "$DATA_ROOT/Outputs" ]
```
Comment (as2388) [inline run_scripts/3_generate_outputs.sh:36]: Please add a comment explaining why this directory needs to be deleted

File: run_scripts/5_automated_analysis.sh [hunk 3]
```diff
@@ -28,7 +28,12 @@ GOOGLE_CLOUD_CREDENTIALS_FILE_PATH=$2
 PIPELINE_CONFIGURATION_FILE_PATH=$3
 DATA_ROOT=$4
 
-mkdir -p "$DATA_ROOT/Outputs"
+if [ -d "$DATA_ROOT/Outputs/Automated Analysis" ]
+then
+    rm -r "$DATA_ROOT/Outputs/Automated Analysis"
+fi
+
+mkdir -p "$DATA_ROOT/Outputs/Automated Analysis"
```
Comment (as2388) [inline run_scripts/5_automated_analysis.sh:37]: same here

**Review by as2388 — APPROVED** [review]

File: run_scripts/3_generate_outputs.sh (hunk 2 above)
Comment (IsaackMwenda) [reply run_scripts/3_generate_outputs.sh:36]: updated thanks.

File: run_scripts/5_automated_analysis.sh (hunk 3 above)
Comment (IsaackMwenda) [reply run_scripts/5_automated_analysis.sh:37]: updated thanks

**Pull request merged**